src/merge-queue.ts

```diff
@@ -33,7 +33,7 @@ export class MergeQueue extends Component {
     super(project);
 
     const autoMerge = options.autoMerge ?? true;
-    const mergeBranch = options.mergeBranch ?? 'main';
+    const mergeBranch = options.mergeBranch ?? options.autoMergeOptions?.mergeBranch ?? 'main';
```

Comment:
curious why you weren't able to add the branch name you want into `options.mergeBranch`? When would there be a scenario where you want to keep `options.mergeBranch` undefined and instead put it into `options.autoMergeOptions?.mergeBranch`?

Reply:
Yeah, I don't like this very much, either, but another option I considered was to add a `mergeBranch` to either `CdkCommonOptions` or `NodeProjectOptions` and pass it to `MergeQueue` like this:

```ts
if (opts.enablePRAutoMerge) {  
  new MergeQueue(project, {  
    autoMergeOptions: opts.ghAutoMergeOptions,  
    mergeBranch: opts.mergeBranch,
  });  
}
```

But `mergeBranch` is not a common option. It's specific to auto merge. So, I decided to add it to `AutoMergeOptions`. And then slightly change the code above to:

```ts
if (opts.enablePRAutoMerge) {  
  new MergeQueue(project, {  
    autoMergeOptions: opts.ghAutoMergeOptions,  
    mergeBranch: opts.ghAutoMergeOptions?.mergeBranch,
  });  
}
```

which is fine, but since we are already passing a `autoMergeOptions`to the `MergeQueue`'s constructor, why pass this particular property separate from the rest?

In any case, this is not a perfect option, either, because `AutoMergeOptions` was intended to be used by the `AutoMerge` construct, which only does part of the work of setting up the auto-merge queue. But I think it's a reasonable compromise.

Reply:
`AutoMerge` is a separate component that should be able to exists on its own. I don't think this is the right compromise here.